src/OrbitBase/File.cpp

```diff
@@ -94,4 +112,25 @@ ErrorMessageOr<size_t> ReadFully(const unique_fd& fd, void* buffer, size_t size)
   return size - bytes_left;
 }
 
+ErrorMessageOr<size_t> ReadFullyAtOffset(const unique_fd& fd, void* buffer, size_t size,
+                                         off_t offset) {
+  uint8_t* current_position = static_cast<uint8_t*>(buffer);
```

Comment:
I find it inconsistent that `uint8_t` is used here but `char` is used in `WriteFullyAtOffset`.

Reply:
Yes, I just copied that from the ReadFully / WriteFully methods. 

A much worse inconsistency is that WriteFully takes a std::string_view as an input and the other methods taks `void*`. I guess a absl::Span would be the right solution but I haven't put any thought into this yet.

Reply:
will leave it as is for now